src/main/scala/ScalaModulePlugin.scala

```diff
@@ -11,9 +11,13 @@ import sbt.librarymanagement.ivy.IvyDependencyResolution
 import sbt.librarymanagement.{ UnresolvedWarningConfiguration, UpdateConfiguration }
 
 object ScalaModulePlugin extends AutoPlugin {
-  val repoName            = settingKey[String]("The name of the repository under github.com/scala/.")
-  val mimaPreviousVersion = settingKey[Option[String]]("The version of this module to compare against when running MiMa.")
-  val scalaVersionsByJvm  = settingKey[Map[Int, List[(String, Boolean)]]]("For a Java major version (6, 8, 9), a list of a Scala version and a flag indicating whether to use this combination for publishing.")
+  object autoImport {
+    val repoName = settingKey[String]("The name of the repository under github.com/scala/.")
+    val mimaPreviousVersion = settingKey[Option[String]]("The version of this module to compare against when running MiMa.")
```

Comment:
`mima` is a prefix used by MiMa plugin, so this especially seems bad, as it would cause a conflict if MiMa decides to add this feature.